### PR TITLE
Javascript/javascript2/week2

### PR DESCRIPTION
**Pull request by LucyChyzhova**

### CODE REVIEW

File: javascript/javascript2/week2/1.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+function multByTwo(number) {
+  return number * 2;
+}
+const multByTwoArray = newNumbersArray.map(multByTwo);
+console.log(multByTwoArray);
+
+//// OR
+
+const newNumbersArray2 = numbers.filter(n => n % 2 === 0);
```
Comment (florinmaticu) [inline javascript/javascript2/week2/1.js:30]: good. remember you can chain this functions like this. can you explain why the code bellow works?
`result = numbers.filter(number => {return number%2 != 0}).map( number => {return 2*number})
`

Comment (LucyChyzhova) [reply javascript/javascript2/week2/1.js:30]: Hi Florin. 
result = numbers
              .filter(number => {return number%2 != 0})
              .map( number => {return 2*number})
This code do the same, first step it filtrs and takes just numbers which can be divide by 2 and do not equal 0. And then this result ( each number) multiplys by 2

Comment (florinmaticu) [reply javascript/javascript2/week2/1.js:30]: Your code is correct,yes. I was pointing out that you can make it shorter, without extra variables

Comment (LucyChyzhova) [reply javascript/javascript2/week2/1.js:30]: yes, yes, it's definitely shorter)) thank you!

**Pull request merged**